### PR TITLE
Book corrections to allow http api access for docker use

### PR DESCRIPTION
**Pull request by AgeManning**
User's on discord has issues with docker and hitting the http api. 

The docs currently are misleading and will confuse users.

This updates the docs around docker allowing users to hit the http api server externally.

### CODE REVIEW

**Comment by paulhauner** [comment]
On second thoughts, now your beacon node is open to the world.

Perhaps we should add a note that this is the case, or alternative use bind mode?

**Comment by AgeManning** [comment]
See fix

File: book/src/docker.md [hunk 1]
```diff
@@ -81,5 +81,5 @@ If you use the `--http` flag you may also want to expose the HTTP port with `-p
 5052:5052`.
```
Comment (paulhauner) [inline book/src/docker.md]: This bit should probably be updated too

**Comment by AgeManning** [comment]
Updated, once you change the status I'll merge

**Pull request merged**